devito/archinfo.py

```diff
@@ -64,12 +66,16 @@ def get_cpu_brand():
 
         cpu_info['flags'] = get_cpu_flags()
         cpu_info['brand'] = get_cpu_brand()
+        if cpu_info['brand'] is None:
+            cpu_info['brand'] = cpuinfo.get_cpu_info().get('raw_arch_string')
```

Comment:
Actually, this should be moved [here](https://github.com/devitocodes/devito/blob/master/devito/archinfo.py#L63) in place of the `return None` right?